src/fsharp/FSharp.Build/FSharp.Build.fsproj

```diff
@@ -6,6 +6,7 @@
     <OutputType>Library</OutputType>
     <TargetFramework Condition="'$(Configuration)' != 'Proto'">netstandard2.0</TargetFramework>
     <TargetFrameworks Condition="'$(Configuration)' == 'Proto'">netstandard2.0</TargetFrameworks>
```

Comment:
Drop these two lines since it's covered by below.

Reply:
Huh !!!! go figure ... I will be super glad to drop those lines.